src/Popover/Popover.js

```diff
@@ -130,6 +130,7 @@ class Popover extends Component {
         });
       } else {
         if (nextProps.animated) {
+          if (this.state.closing) return;
```

Comment:
I'm wondering, the `setState` is asynchronous, wouldn't be better to use a class property to no trigger multiple timer. E.g we could be using `this.timeout !== null`.